### ISSUE TITLE
Revamping 'What's new' window for seemless experience

### ISSUE THREAD

**Issue by SparshKat** [mozilla/hubs#2002]
**Problem**
As a new contributor to this project, first thing i noticed was 'What's new' tab of the main (hubs) site. Apart from slow loading of the window, 'What's new' displays information which is not entirely necessary in one go. Upon scrolling down the what's new window, it keeps on expanding and it's not easy to pin point to one information if one wants to retrieve. This window should be a main hub for providing with information necessary to the user along with describing things in an efficient manner but it's not able to to do so and it doesn't even look very good due to simple static webpage.

**Solution**
The timeline of the updates listed in the main window should be segregated in blocks of time. Like one square with label of the year and in it a label of month and so on. Upon opening all of them, should be the updates listed with specific dates in the specified month. The upper space should be occupied with a huge moving jumbotron showing the latest 5 updates done. 

**More things which can be added**
As a beginner, i had a lot of difficulties and had to try out things to see how the creation of room works, but still haven't figured everything out. If a beginner's guide would be given, then it would be a huge help for all of the people trying out the website rather than losing their patience and giving up soon.

**Comment by gfodor** [comment]
Thanks for the feedback! We are currently working on better documentation to help users get up to speed on hubs. For the What's New page, we started with the simple page you see now but would welcome additional designs to consider. Right now I don't expect our team to prioritize overhauling the page currently, but with a good design in-hand that might change since it'd reduce the work involved significantly.

**Comment by SparshKat** [comment]
Yes i didn't want to prioritise this task as such but wanted to make it better so that people visiting for the first time would have a smoother flow of work. Plus i'd love to suggest a simple design for it (kinda what i have mentioned) i will surely love to provide a design theme for it and would contribute to it alongside. Or if i can contribute to this later on that would work?

**Comment by SparshKat** [comment]
I'd love to work for it sooner or later so as to make the project better with my contributions so hope you assign it to me later on.

**Comment by gfodor** [comment]
Thank you for offering! If you would like to create a design for us to review, we'd be happy to consider it!

**Comment by joshmarinacci** [comment]
Hi @SparshKat did you ever work on this?
